index.html

```diff
@@ -52,7 +52,271 @@ <h3>
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+
+          <li class="recent-works">
+            <h3>
+              Profesional Art Printing Data
+            </h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="technologies-used-recent-works">
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+
+          <li class="recent-works">
+            <h3>
+              Profesional Art Printing Data
+            </h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="technologies-used-recent-works">
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+
+          <li class="recent-works">
+            <h3>
+              Profesional Art Printing Data
+            </h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="technologies-used-recent-works">
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+        </ul>
+      </section>
+
+
+
+      <section class="about-me-container">
+        <div class="about-me-title">
+          <p>About Me</p>
+        </div>
+        <p> 
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+  
+        <button type="submit" class="see-project-button">GET MY REUSME</button>
+
+        <div class="spacer">.</div>
+
+        <div class="about-me-list">
```

Comment:
Your languages section also does not match the Figma template of your choosing. Kindly fix it and then ask for a re-review.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178111756-8f2501bb-c960-440b-804d-19c3dc5b3567.png) | ![image](https://user-images.githubusercontent.com/33268927/178111779-61bb1bef-4b96-4746-aa66-b444d9afcd8b.png) |